packages/protobuf/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 1.0.14-beta.1
+
+-   feat(suite): Add adding a new shamir group into an existing setup (e9a0425183)
+-   feat(connect): Conway certificates (ab003ce04f)
+-   fix(suite): make default 1:1 for adding backup from settings + sync protobuffs (b08d5602fb)
```

Comment:
similar to this https://github.com/trezor/trezor-suite/pull/12759/files#r1629462181

lets please @peter-sanderson  make commits atomic when we are handlign public packages as we use these commit messages to generate changelogs. 
also `suite` keyword shoudn't appear here. 

the other way around might be removing the `(package)` part? would that help automate things @karliatto ? or maybe is it better to write nicer commit messages and edit them occasionaly if we asses they are not good enough?